docs/core/knowledge-bases.rst

```diff
@@ -0,0 +1,383 @@
+:desc: Leverage information from knowledge bases inside conversations using ActionQueryKnowledgeBase
+       in open source bot framework Rasa.
+
+.. _knowledge_bases:
+
+Knowledge Base Actions
+======================
+
+.. edit-link::
+
+.. warning::
+   This feature is experimental.
+
+.. contents::
+   :local:
+
+A lot of users want to obtain detailed information about certain objects, such as restaurants or hotels, during a conversation.
```

Comment:
I think we can make this easier to understand by motivating with an example. Otherwise this sounds very abstract